src/arviz_plots/style.py

```diff
@@ -39,53 +41,51 @@ def use(name):
 
 def available():
     """List available styles."""
+    styles = {}
+
     try:
         import matplotlib.pyplot as plt
 
-        print("Matplotlib styles:")
-        print(plt.style.available)
-        print()
+        styles["matplotlib"] = plt.style.available
     except ImportError:
         pass
 
     try:
         import plotly.io as pio
 
-        print("Plotly templates:")
-        print(list(pio.templates))
+        styles["plotly"] = list(pio.templates)
     except ImportError:
         pass
 
+    return styles
 
-def get(name, backend="matplotlib"):
+
+def get(name, backend=None):
     """Get the style/template with the given name.
 
     Parameters
     ----------
     name : str
         Name of the style/template to get.
     backend : str
-        Name of the backend to use. Options are 'matplotlib' (default) and 'plotly'.
+        Name of the backend to use. Options are 'matplotlib' and 'plotly'.
+        Defaults to ``rcParams["plot.backend"]``.
     """
-    if backend not in ["matplotlib", "plotly"]:
+    if backend is None:
+        backend = rcParams["plot.backend"]
+    elif backend not in ["matplotlib", "plotly"]:
```

Comment:
```suggestion
    if backend not in ["matplotlib", "plotly"]:
```

I think it should still fail with this error if running wih bakcend=None and plot.backend rcparam set to bokeh